setup.py

```diff
@@ -28,6 +28,12 @@
     keywords="django download asynchronous async celery",
     packages=["async_downloads"],
     include_package_data=True,
-    install_requires=['Django>=2.0', 'celery>=4.2.1', 'pathvalidate>=2.3.0'],
+    install_requires=[
+        "Django>=2.0",
+        "celery>=4.2.1",
+        "pathvalidate>=2.3.0",
+        "channels>=3.0.4",
+        "channels_redis>=3.3.1",
```

Comment:
Does it specifically need to be these versions or can we loosen it a bit to something like (this is without looking anything up):
```suggestion
        "channels>=3.0",
        "channels_redis>=3.0",
```

Reply:
533d04f